crates/valence_java_string/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "valence_java_string"
```

I've got the `java_string` name reserved on crates.io :)

```suggestion
name = "java_string"
```